src/main/java/frc/robot/subsystems/Elevator.kt

```diff
@@ -42,9 +42,9 @@ object Elevator : SubsystemBase("Elevator") {
                 }
             )
 
-    val isDown: BooleanSupplier = BooleanSupplier { position < 3.inches }
+    val isStowed: BooleanSupplier = BooleanSupplier { position < IS_STOWED_THRESHOLD }
```

Comment:
I think it makes more sense to just have this be a normal boolean (with a getter), and when it needs to be a supplier, wrap in a lambda

Reply:
I would also just use raw lambdas, and not wrap in the BooleanSupplier class. You can set the type as `() -> Boolean` if needed, but it'll infer